packages/patternlab/styleguide/source/assets/scss/index.scss

```diff
@@ -161,6 +161,7 @@
 @use "03-organisms/header-hamburger";
 @use "03-organisms/header-mixed";
 @use "03-organisms/header-alert";
```

Comment:
Since we are fulling replacing header-alert with header alerts, we should just deprecate and clean up header-alert